xla/stream_executor/gpu/gpu_stream.cc

```diff
@@ -15,13 +15,22 @@ limitations under the License.
 
 #include "xla/stream_executor/gpu/gpu_stream.h"
 
+#include "tsl/platform/status.h"
 #include "xla/stream_executor/gpu/gpu_executor.h"
 #include "xla/stream_executor/stream.h"
-#include "tsl/platform/status.h"
 
 namespace stream_executor {
 namespace gpu {
 
+bool GetStreamPriorityRange(int* lowest, int* highest) {
+  CUresult res = cuCtxGetStreamPriorityRange(lowest, highest);
```

Comment:
Unfortunately, I think this has to be pushed down further into GpuDriver I think as AMD ROCm build will fail otherwise.

Reply:
I see, i though this class was specific to CUDA since I saw a few references to CUDA in the header. I will move it to gpuDriver then.